pkg/manager/member/ticdc_scaler.go

```diff
@@ -162,6 +206,82 @@ func gracefulShutdownTiCDC(
 	return nil
 }
 
+const ticdcCrossUpgradeVersion = "6.3.0"
+
+// A TiCDC can graceful upgrade when we are performing reload or the TiCDC pod
+// version is greater or equal to v6.3.0.
+func isTiCDCPodSupportGracefulUpgrade(
+	tc *v1alpha1.TidbCluster,
+	cdcCtl controller.TiCDCControlInterface,
+	podCtl controller.PodControlInterface,
+	pod *corev1.Pod,
+	ordinal int32,
+	action string,
+) (bool, error) {
+	isTimeout, err := checkTiCDCGracefulShutdownTimeout(tc, podCtl, pod, action)
+	if err != nil {
+		return false, err
+	}
+	if isTimeout {
+		return false, nil
+	}
+	status, err := cdcCtl.GetStatus(tc, ordinal)
+	if err != nil {
+		return false, controller.RequeueErrorf(
+			"ticdc.%s: cluster %s/%s fail to get TiCDC status, error: %v",
+			action, tc.GetNamespace(), tc.GetName(), err)
+	}
+	podVersion := status.Version
+	currentVersion := tc.Spec.TiCDC.Version
+	if currentVersion == nil {
+		currentVersion = &tc.Spec.Version
+	}
+
```

Comment:
If the client use the `image` field, the version is empty. How about adding a field `TiCDCVersion()` to parse version, like `TiFlashVersion`.
```go
// TiFlashVersion returns the image version used by TiFlash.
//
// If TiFlash isn't specified, return empty string.
func (tc *TidbCluster) TiFlashVersion() string {
	if tc.Spec.TiFlash == nil {
		return ""
	}

	image := tc.TiFlashImage()
	colonIdx := strings.LastIndexByte(image, ':')
	if colonIdx >= 0 {
		return image[colonIdx+1:]
	}

	return "latest"
}
```

In the future, we just need modify this function when we remove the `image` filed.